src/main/main.c

```diff
@@ -234,7 +234,7 @@ void init(void)
     servoMixerInit(masterConfig.customServoMixer);
 #endif
 
-#ifdef SPEKTRUM_BIND
+#if defined(SERIAL_RX) && defined(SPEKTRUM_BIND)
```

Comment:
This is a hen&egg problem with the preprocessor, not having deferred evaluation as make has. 
We could re-order the includes, target configuration before common. but that would lead to other disasters. 
Or, re-do the build config handling. Move all conditional settings and all their dependency conditions into "target.mk" and a "common.mk". Config settings could then be added to compiler command line or a temporary buildconfig.h file. One other major advantage with this would be that the Makefile could contain conditionals per function or feature. As of today this is not possible when the feature defs are in the header files. Major drawback.